inventory/rh-sso/group_vars/all.yml

```diff
@@ -14,3 +14,13 @@ rhsm_repos:
 rhsm_username: "{{ lookup('env', 'RHSM_USER' )}}"
 rhsm_password: "{{ lookup('env', 'RHSM_PASSWD' )}}"
 rhsm_pool: "{{ lookup('env', 'RHSM_POOL' )}}"
+
+# User management variables
+# Username must be lowercase
+sso_user_params:
```

Comment:
Think this might have gotten lost in the shuffle, but we want to have the `users` list in the format that is listed in this README: https://github.com/redhat-cop/infra-ansible/tree/master/roles/identity-management

I think we can also bump this out to the `identity_hosts.yml` file under `group_vars` as this only pertains to roles used there.